OData/src/System.Web.OData/OData/CustomAggregateMethodAnnotation.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections.Generic;
+using System.Reflection;
+
+
+namespace System.Web.OData
+{
+    /// <summary>
+    /// Allows client to tell OData which are the custom aggregation methods defined.
+    /// In order to do it it must receive a methodToken - that is the full identifier
+    /// of the method in the OData URL - and an IDictionary that maps the input type
+    /// of the aggregation method to it's MethodInfo.
+    /// </summary>
+    public class CustomAggregateMethodAnnotation 
+    {
+
+        Dictionary<string, IDictionary<Type, MethodInfo>> TokenToMethodWrapper;
+
+        /// <summary>
+        /// CustomAggregateMethodAnnotation simple constructor.
+        /// </summary>
+        public CustomAggregateMethodAnnotation()
+        {
+            TokenToMethodWrapper = new Dictionary<string, IDictionary<Type, MethodInfo>>();
```

Comment:
You could to that initialization in the same line as definition of the property.